v1/authorization/jwt.ts

```diff
@@ -0,0 +1,36 @@
+import { Static } from '@sinclair/typebox'
+import { nanoid } from 'nanoid';
+import { Admin, Publisher } from '../api/schemas.js'
+
+export type Creator = "system" | string;
```

Comment:
We should expose consts for the `system` `admin` and `publisher` strings for use elsewhere.